library/pom.xml

```diff
@@ -67,13 +67,13 @@
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>7.4.0</version>
+            <version>7.6.1</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
-            <version>1.7.30</version>
+            <version>1.7.36</version>
```

Comment:
There is 2.0 as stable these days.

Reply:
Seen it, but it can create problems with other libraries that we use, which depends on 1.7.36